TrkReco/src/MergeHelices_module.cc

```diff
@@ -183,13 +196,46 @@ namespace mu2e {
     nh2 = shiv2.size();
     nover = countOverlaps(shiv1,shiv2);
   }
-
+	
+  // The chisquared calculation method used here is 
+  // taken from the LsqSums approach followed in CalPatRec
+  void MergeHelices::findchisq(art::Event const& evt,
+      HelixSeed const& h1,float& chixy, float& chizphi) {
+    ::LsqSums4 sxy;
+    ::LsqSums4 szphi;
+    for(size_t ihit=0;ihit < h1.hits().size(); ihit++) {
+      auto const& hh = h1.hits()[ihit];
+      float transErr = 5./sqrt(12.);
```

Comment:
This number doesn't change, so calculate it in the constructor.  The value 5 here (and 1.1 and 0.75 below) are algorithmically tuned values, they should be made into class variables and initialized via fhicl parameter.  If there's a physical explanation what they mean or why they are set to these values (maybe Pasha knows) that shoudl be explained.  Note that the code should not take default values; they should be set in the appropriate prolog.fcl file.

Reply:
Hi Dave, so I should set the deltaNhits, XY and Phi weights (1.1 and 0.75) in the prolog.fcl right ? Shall make the changes ASAP